virt/monitoring/virt-runbooks.adoc

```diff
@@ -6,119 +6,296 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-
 :!virt-runbooks:
-You can use the procedures in these runbooks to diagnose and resolve issues that trigger {VirtProductName} xref:../../observability/monitoring/managing-alerts.adoc#managing-alerts[alerts].
+Runbooks for the {VirtProductName} Operator are maintained in the link:https://github.com/openshift/runbooks/tree/master/alerts/openshift-virtualization-operator[openshift/runbooks] Git repository and can be viewed on GitHub. You can use the procedures in these runbooks to diagnose and resolve issues that trigger {VirtProductName} xref:../../observability/monitoring/managing-alerts.adoc#managing-alerts[alerts].
 
 {VirtProductName} alerts are displayed in the *Virtualization* -> *Overview* tab in the web console.
 
+// IMPORTANT: these IDs are meant to exactly mirror the original module IDs //
+// changing them will break links in the actual alerts //
+
+[id="virt-runbook-CDIDataImportCronOutdated"]
```

Comment:
@ousleyp **Global comment for all Level 1 or lower subsections in an assembly**: Suggest adding a _{context} variable to the end of anchor IDs.

As par RH SSG, for any Level 1 or lower subsections in an assembly (for example, ==, ===, etc.), add a _{context} variable to the end of anchor IDs:

Example anchor ID for a Level 1 heading (subsection) of an assembly file
[id="editing-alerts_{context}"]
== Editing alerts

Reply:
I don't think I can do that in this case, because the anchors need to match the pre-existing anchors exactly. I don't think there will be a problem with these being reused somewhere else due to their nature. However, they all start with "virt-runbook-," so it's _almost_ like they have the context statement as part of the ID. :)

Reply:
It _is_ odd that there wasn't `_{context}` in the module anchors in any of the deleted modules... except I just found that it's in the IDs in 4.12.